continuous_integration/hdfs/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:trusty
+
+RUN apt-get update && \
+    apt-get install -y -q curl bzip2 git && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install CDH5 in a single node: Pseudo Distributed
+# Docs: http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cdh_qs_yarn_pseudo.html
+ADD docker-files/cloudera.pref /etc/apt/preferences.d/cloudera.pref
+RUN curl -s http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key | apt-key add - && \
+    echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list && \
+    echo 'deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' >> /etc/apt/sources.list.d/cloudera.list && \
+    apt-get update && \
+    apt-get install -y -q openjdk-7-jre-headless hadoop-conf-pseudo && \
+    sudo -u hdfs hdfs namenode -format -force && \
+    for x in `cd /etc/init.d ; ls hadoop-hdfs-*` ; do sudo service $x start ; done && \
+    bash /usr/lib/hadoop/libexec/init-hdfs.sh && \
+    rm -rf /var/lib/apt/lists/*
+
+# Needed for hdfs3
+ENV LIBHDFS3_CONF /etc/hadoop/conf/hdfs-site.xml
```

Comment:
Did you find this is really needed? The hdfs3 import should set the value upon import, before loading the .so.

Reply:
This was copied from before. I'd be hesitant to change here, as this code was fine then.